packages/mdc-ripple/README.md

```diff
@@ -121,6 +95,8 @@ When a ripple is successfully initialized on an element, it dynamically adds a `
 
 This code sets up `.surface` with the correct css variables as well as `will-change` properties to support the ripple. It then dynamically generates the correct selectors such that the surface's `::before` element functions as a background ripple, and the surface's `::after` element functions as a foreground ripple.
 
+When a ripple is successfully initialized on an element, it dynamically adds a `mdc-ripple-upgraded` class to that element. If ripple is not initialized but Sass misins are included within our surface, the ripple will still work, but it would use a simpler, CSS-Only implementation which relies on `::hover`, `::active`, and `::focus`.
```

Comment:
nit: `:hover`, `:active`, and `:focus`, not double-colon. Double-colons are for pseudo-_elements_, whereas single-colons are for pseudo-_classes_